test/yubico_test.sh

```diff
@@ -0,0 +1,29 @@
+# Basic test
+# Requires dfu-utils to be installed.
```

Comment:
libu2f-host instead of dfu-utils?

Reply:
Package "u2f-host" on debian.  Pushed this change